src/shared/netif-naming-scheme.h

```diff
@@ -53,6 +53,7 @@ typedef enum NamingSchemeFlags {
         NAMING_V252 = NAMING_V251 | NAMING_DEVICETREE_ALIASES,
         NAMING_RHEL_9_0 = NAMING_V250 | NAMING_BRIDGE_MULTIFUNCTION_SLOT,
         NAMING_RHEL_9_1 = NAMING_RHEL_9_0,
+        NAMING_RHEL_9_2 = NAMING_RHEL_9_0,
```

Comment:
Eh, this should have been set to `NAMING_V252`. But I don't think it's worth changing it now, as the "devicetree aliases" thingy doesn't look very important.

Reply:
I believe that we have discussed it with @msekletar and concluded that we don't want to change the naming scheme mid-major release. But I might be mistaken.

Reply:
`rhel-9.0` shall remain _the default_ scheme, sure. But subsequent schemes should cover changes. That's the whole point of their existence.